### PR TITLE
Update README.md for GLEAM

### PR DESCRIPTION
**Pull request by qchiujunhao**

### CODE REVIEW

**Review by paulocilasjr — COMMENTED** [review]
On line 6 - where it reads: "suite of machine learning tools for the [Galaxy](https://galaxyproject.org/) platform", I always wonder if we should point to galaxy project page or directly to the galaxy server (https://usegalaxy.org/). 

On line 12 - include "best practices". 
Also I think we can bring this to the first point:
## Features
- Modern best practices for machine learning
- Reproducible and scalable workflows
- Machine learning support for diverse data types: tabular, image, text, categorical, and more
- Deep learning via Ludwig and automated ML via PyCaret
- Easy installation in Galaxy via XML wrappers
- Auto-generated visual reports

Suggestion for the tools descriptions:

### 1. TabularLearner
- input files: csv, tsv
- tasks: classification and regression models
- Models available: Logistic Regression, SVM - Linear Karnel, Ada Boost Classifier, Linear regression, Elastic Net, Lasso Least Angle Regression and more...
- Output: Interactive HTML reports (tables, graphs and figures) to evaluate performance and extract feature importance

### 2. ImageLearner
- input files: Zip file with images  and csv with metadata
- Tasks: classification
- Models available: ResNet, EfficientNet, VGG, Shufflenet, Vit, AlexNet and More...
- Output: Ludwig_model file, a report in the form of an HTML file (with learning curves, confusion matrices, and etc...), and a collection of CSV/json/png files containing the predictions, experiment stats and visualizations.

### 3. Galaxy-Ludwig
General-purpose interface to Ludwig's full machine learning capabilities.

- Train and evaluate models on structured input (tabular, image, text, etc.)
- Expose Ludwig’s flexible configuration system
- Ideal for users needing advanced model customization

### 4. Galaxy-Digital Pathology Processing
set of three specialized tools designed to transforms raw, large pathology images into a structured format, enabling the application of best practices for model development and ensuring data readiness for robust and efficient training.
1) Image Tiler: Accepts .svs image format, which is the most common proprietary format for digital pathology whole slide images.

2) Embedding Extractor:Leverages pre-trained models from the TorchVision foundation models for feature extraction (for example, ResNet50, EfficientNet_B0, DenseNet121).

3) Multiple Instance Learning (MIL) Bag Processor:
Facilitates the aggregation of embeddings from individual image tiles into "bags" using various pooling techniques (such as Max Pooling or Attention Pooling).

**Review by paulocilasjr — APPROVED** [review]
Looks good.
